includes/class-comment.php

```diff
@@ -340,6 +341,31 @@ public static function comment_class( $classes, $css_class, $comment_id ) {
 		return $classes;
 	}
 
+	/**
+	 * Filters the comment feed where clause to exclude ActivityPub comment types.
+	 *
+	 * @param string $where The WHERE clause for the comment feed query.
+	 *
+	 * @return string The modified WHERE clause.
+	 */
+	public static function comment_feed_where( $where ) {
+		$comment_type = \get_query_var( 'type' );
+
+		if ( 'all' === $comment_type ) {
+			return $where;
+		}
+
+		$comment_types = self::get_comment_type_slugs();
+
+		if ( in_array( $comment_type, $comment_types, true ) ) {
+			$where .= " AND comment_type = '" . esc_sql( $comment_type ) . "'";
+		} else {
+			$where .= " AND comment_type = 'comment'";
```

Comment:
Rather than making decisions for comment types no registered by us, can we use a NOT IN and only exclude `$comment_types`?